distribution/build.sh

```diff
@@ -96,7 +96,7 @@ echo "\$JAVA_OPTS += \"-Dkeycloak.import=\$pwd\\\..\\\realm\\\alfresco-realm.jso
 # zip the confiured distro
 zip -r $DISTRIBUTION_NAME.zip $DISTRIBUTION_NAME
 openssl md5 $DISTRIBUTION_NAME.zip > $DISTRIBUTION_NAME.md5
-rm -rf $DISTRIBUTION_NAME
+#rm -rf $DISTRIBUTION_NAME
```

Comment:
If we don't ever need this let's remove the line, not comment it out.

Reply:
fixed under  [7d2afd](https://github.com/Alfresco/alfresco-identity-service/pull/106/commits/7d2afddec1d32794b2804fed515f2a053c4b1e33)